client/src/pages/Review/DocumentPanel.tsx

```diff
@@ -1,28 +1,34 @@
-import { SetStateAction, useState } from "react";
+import { SetStateAction, useState, LegacyRef } from "react";
 import { Document, Page } from "react-pdf/dist/esm/entry.vite";
 import "react-pdf/dist/esm/Page/AnnotationLayer.css";
 import "react-pdf/dist/esm/Page/TextLayer.css";
+import { useMeasure } from "react-use";
 
 export interface DocumentPanelProps {
   file: File;
 }
 
 const DocumentPanel = ({ file }: DocumentPanelProps) => {
   const [numPages, setNumPages] = useState(0);
+  const [myRef, { width }] = useMeasure();
 
   return (
-    <Document
-      file={file}
-      onLoadSuccess={(pdfInfo: { numPages: SetStateAction<number>; }) => setNumPages(pdfInfo.numPages)}
-      onLoadError={(e: unknown) => alert(e)}
-    >
-      {Array.from(new Array(numPages), (el, index) => (
-        <div key={index}>
-          <Page pageNumber={index + 1} />
-          <hr className="border-2 border-gray-200" />
-        </div>
-      ))}
-    </Document>
+    <div ref={myRef as LegacyRef<HTMLDivElement>}>
+      <Document
+        file={file}
+        onLoadSuccess={(pdfInfo: { numPages: SetStateAction<number> }) =>
+          setNumPages(pdfInfo.numPages)
+        }
+        onLoadError={(e: unknown) => alert(e)}
+      >
+        {Array.from(new Array(numPages), (el, index) => (
+          <div key={index}>
+            <Page pageNumber={index + 1} width={width} />
```

Comment:
Annoyingly, the only way to limit the width of the pdf pages is to use the `width` prop on `Page` which is pixels only, necessitating the useMeasure hook. It won't take a className and won't react to being wrapped in smaller things